### ISSUE TITLE
Cannot get the HUD to appear

### ISSUE THREAD

**Issue by ciel** [avanderhoorn/Glimpse.Prototype#101]
I've followed the installation instructions to the letter for Glimpse v2 beta1, and I get..... !

_nothing_. A lot of nothing. I'm not even sure how to begin diagnosing this. I see no errors, no reports of problems, nothing. Running ASP.NET 5 RC1.

I don't even know what information you need. I'm fairly lost, unfortunately.

**Comment by KoalaBear84** [comment]
Yes. Me too, I'm also lost. Also see this thread. Currently also no response. I also tried the manual way, but failed.

https://github.com/Glimpse/Glimpse.Prototype/commit/0a28805298e7002ed1124c53cd914519b8248c0f#commitcomment-15430857

**Comment by Software-Princess** [comment]
Did you try adding the javascript file to the html page?

`<script src='/Glimpse/Browser/Agent' id='glimpse' ></script>`

**Comment by KoalaBear84** [comment]
Hmm. Never saw that piece of code anywhere.

For now I will wait until there is a published version onto a NuGet source. Or is there one already for ASP.NET Core MVC?
